bin/ember-template-lint.js

```diff
@@ -165,7 +154,18 @@ function parseArgv(_argv) {
         boolean: true,
       },
       'print-pending': {
-        describe: 'Print list of formatted rules for use with `pending` in config file',
+        describe:
+          'Print list of formatted rules for use with `pending` in config file (deprecated)',
+        boolean: true,
+      },
+      'update-todo': {
+        describe: 'Update list of linting todos by transforming lint errors to todos',
+        default: false,
+        boolean: true,
+      },
+      'include-todo': {
+        describe: 'Include todos in the results (todos are hidden by default)',
```

Comment:
```suggestion
        describe: 'Include todos in the results',
```

we don't have to say the "hidden by default" thing, now that we added `default: false`

Reply:
Removed.